lib/flagutil/secret_test.go

```diff
@@ -0,0 +1,21 @@
+package flagutil
+
+import "testing"
+
+func TestApplySecretFlags(t *testing.T) {
+	secretFlags = make(map[string]bool)
```

Comment:
Why this initialisation is needed?
As far as I can see, value will be initialised with the default - https://github.com/VictoriaMetrics/VictoriaMetrics/blob/71c4477c6bf8dbfda215882bcea8a042218ba6e6/lib/flagutil/secret.go#L18